dask/highlevelgraph.py

```diff
@@ -500,6 +494,17 @@ def _repr_html_(self, layer_index="", highlevelgraph_key=""):
         else:
             shortname = self.__class__.__name__
 
+        svg_repr = ""
+        if (
+            self.collection_annotations
+            and self.collection_annotations.get("type") == "dask.array.core.Array"
+        ):
+            chunks = self.collection_annotations.get("chunks")
+            if chunks:
+                from .array.svg import svg
```

Comment:
We talked about this import placement previously, and I think this is probably the best way to leave it

Reply:
:heavy_check_mark: